packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -2033,6 +2033,64 @@ describe('KeyringController', () => {
         );
       });
     });
+
+    describe('getKeyringsByType', () => {
+      beforeEach(() => {
+        jest
+          .spyOn(KeyringController.prototype, 'getKeyringsByType')
+          .mockReturnValue([
+            {
+              type: 'HD Key Tree',
+              accounts: ['0x1234'],
+            },
+          ]);
+      });
+
+      it('should return correct keyring by type', async () => {
+        await withController(async ({ controller, messenger }) => {
+          messenger.call('KeyringController:getKeyringsByType', 'HD Key Tree');
+
+          expect(controller.getKeyringsByType).toHaveBeenCalledWith(
+            'HD Key Tree',
+          );
+        });
+      });
+    });
+
+    describe('getKeyringForAccount', () => {
+      beforeEach(() => {
+        jest
+          .spyOn(KeyringController.prototype, 'getKeyringForAccount')
+          .mockResolvedValue({
+            type: 'HD Key Tree',
+            accounts: ['0x1234'],
+          });
+      });
+
+      it('should return the keyring for the account', async () => {
+        await withController(async ({ controller, messenger }) => {
+          await messenger.call('KeyringController:getKeyringForAccount', '0x0');
+
+          expect(controller.getKeyringForAccount).toHaveBeenCalledWith('0x0');
+        });
+      });
+    });
+
+    describe('getAccounts', () => {
+      beforeEach(() => {
+        jest
+          .spyOn(KeyringController.prototype, 'getAccounts')
+          .mockResolvedValue(['0x1234']);
+      });
```

Comment:
Nit: these `beforeEach` are inside `describe`s that include just one test case:
To make them more readable, can we remove the `beforeEach` and move the `jest.spyOn` into the `test` case itself, or under the upper `describe('actions')` along with the other spies?

Reply:
sure no problem, changed in a3f270c294ac2ad4f5fd081f27505e3d9472634d